doc/rst/users/build.rst

```diff
@@ -7,19 +7,30 @@ Dependencies
 ------------
 
 SCR has several dependencies. A C compiler, MPI, CMake, and pdsh are
-required dependencies. The others are optional, and when they are
+required, as are several component libraries from the ECP VeloC project. The others are optional, and when they are
 not available some features of SCR may not be available.
 
+Required:
+
 * CMake, Version 2.8+
 * Compiler (C, C++, and Fortran)
 * MPI
 * pdsh (https://github.com/grondo/pdsh)
-* DTCMP (optional, used for user-defined directory structure feature)(https://github.com/llnl/dtcmp)
-* LWGRP (optional, used for user-defined directory structure feature) (https://github.com/LLNL/lwgrp)
-* libyogrt (optional, used for determining length of time left in the current allocation) (https://github.com/llnl/libyogrt)
-* MySQL (optional, used for logging SCR activities)
```

Comment:
Did we lose mysql from the list?

Reply:
I thought mysql was gone, but I don't know where I got that impression. Should it be back in? I will look around.

Reply:
I guess there are lots of refs still there. Restoring that line.

Reply:
MySQL is still an optional dependency.  We can just move it down to your new Optional list along with libyogrt.

Reply:
Restored